framework/src/executioners/PicardSolve.C

```diff
@@ -51,6 +51,9 @@ PicardSolve::validParams()
                         "during Picard iterations.  This check is "
                         "performed based on the Master app's nonlinear "
                         "residual.");
+  params.addParam<PostprocessorName>("picard_custom_pp",
+                                     "Postprocessor for custom picard convergence check, "
+                                     "uses 'picard_abs_tol' as tolerance.");
```

Comment:
I suggest to have a separate parameter for pp convergence: `picard_pp_tol`. It is better to check the convergence of a postprocessor with `std::abs((_pp_old - _pp) / _pp) < _picard_pp_tol`, instead of letting the postprocessor converge to zero (I think it is a too special postprocessor for convergence check). In this way, we can monitor any normal postprocessor during Picard iteration, what do you think?